docs/content/docs/dev/table/sqlClient.md

```diff
@@ -169,559 +177,523 @@ Mode "embedded" (default) submits Flink jobs from the local machine.
+[INFO] Submitting SQL update statement to the cluster...
+[INFO] Execute statement in sync mode. Please wait for the execution finish...
+[INFO] Complete execution of the SQL update statement.
+```
 
-Views allow to define virtual tables from SQL queries. The view definition is parsed and validated immediately. However, the actual execution happens when the view is accessed during the submission of a general `INSERT INTO` or `SELECT` statement.
+<span class="label label-danger">Attention</span>  If you want to terminate the job, just type `CTRL-C` to cancel the execution.
 
-Views can either be defined in [environment files](sqlClient.html#environment-files) or within the CLI session.
+### Start a SQL Job from a savepoint
 
-The following example shows how to define multiple views in a file. The views are registered in the order in which they are defined in the environment file. Reference chains such as _view A depends on view B depends on view C_ are supported.
+Flink supports to start the job with specified savepoints. In SQL Client, it's allow to use `SET` command
+to specify the path to the savepoints.
 
-```yaml
-tables:
-  - name: MyTableSource
-    # ...
-  - name: MyRestrictedView
-    type: view
-    query: "SELECT MyField2 FROM MyTableSource"
-  - name: MyComplexView
-    type: view
-    query: >
-      SELECT MyField2 + 42, CAST(MyField1 AS VARCHAR)
-      FROM MyTableSource
-      WHERE MyField2 > 200
+```sql
+Flink SQL> SET execution.savepoint.path=/tmp/flink-savepoints/savepoint-cca7bc-bb1e257f0dab;
+[INFO] Session property has been set.
 ```
 
-Similar to table sources and sinks, views defined in a session environment file have highest precedence.
+When the path is specified, Flink will try to restore the state from the savepoint when execute statements.
 
-Views can also be created within a CLI session using the `CREATE VIEW` statement:
+SQL Client also supports to disable restore from the savepoint by `RESET` command.
 
-```text
-CREATE VIEW MyNewView AS SELECT MyField2 FROM MyTableSource;
+```sql
+Flink SQL> RESET execution.savepoint.path;
+[INFO] Session property has been reset.
 ```
 
-Views created within a CLI session can also be removed again using the `DROP VIEW` statement:
+For more details about creating and managing savepoints, please refer to [Job Lifecycle Management]({{< ref "docs/deployment/cli" >}}#job-lifecycle-management).
 
-```text
-DROP VIEW MyNewView;
+### Define a Custom Job Name
+
+SQL Client supports to name every submitted job through `SET` command.
+
+```sql
+Flink SQL> SET pipeline.name=SqlJob;;
+[INFO] Session property has been set.
 ```
 
-<span class="label label-danger">Attention</span> The definition of views in the CLI is limited to the mentioned syntax above. Defining a schema for views or escaping whitespaces in table names will be supported in future versions.
+<span class="label label-danger">Attention</span> Please set the name before submitting the job.
```

Comment:
I think, in most cases, the default job name works well. We don't need to suggest users specifying job name for every statements (that's quite trivial). 

```suggestion
Because the job name configuration will affect all the following DML statements, you can also use `RESET` command to reset this configuraiton, i.e. use default job names. 
```